src/content/eth2/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Ethereum 2.0 (Eth2)
+description: Everything you need to know about Eth2
+lang: en
+sidebar: true
+---
+
+# Ethereum 2.0 (Eth2)
+
+<Subtitle>Eth2 will give Ethereum the upgrade it needs to scale to serve all of humanity. It’s a multi-phase process but the phase 0 should be with us before the end of 2020.</Subtitle>
+
+Eth2 refers to a major upgrade happening to the Ethereum blockchain. This will make transactions faster, reduce energy consumption and introduce staking rewards. Technically speaking, Ethereum will be moving from proof-of-work to a [proof-of-stake](/en/eth2/proof-of-stake/) blockchain and introducing [shard chains](/en/eth2/shard-chains/). This is a huge change to how Ethereum works but it should bring equally huge benefits.
+
+<InfoBanner emoji=":thinking_face:">
+    <b>Not following?</b> Try reading up on <a href="/en/what-is-ethereum/">Ethereum 1.0</a>
+</InfoBanner>
+
+## Latest status
+
+Eth2 is currently **in testing** so, if you're really bullish, now's the time to help test the latest developments.
+
+This testing phase is the first tangible step in the journey to Eth2. The roadmap will show the ground we still have left to cover.
+
+- [How to get involved](/en/eth2/get-involved/)
```

Comment:
IMO "The roadmap" makes sense to link to first since it's mentioned in the sentence above.

Reply:
agreed